packages/components/src/internal/components/samples/actions.ts

```diff
@@ -327,3 +328,22 @@ export function getGroupedSampleDisplayColumns(
         editColumns,
     };
 }
+
+export function getSelectedItemSamples(selectedItemIds: string[]): Promise<number[]> {
+    return new Promise((resolve, reject) => {
+        getSelectedData('Inventory', 'Item', selectedItemIds, 'RowId, MaterialId', undefined, undefined, undefined)
```

Comment:
do we have constants we can use for the schema name and query name `'Inventory', 'Item'` here?